src/CONST.js

```diff
@@ -402,6 +402,7 @@ const CONST = {
         SHOW_LOADING_SPINNER_DEBOUNCE_TIME: 250,
         TOOLTIP_SENSE: 1000,
         TRIE_INITIALIZATION: 'trie_initialization',
+        COMMENT_LENGTH_DEBOUNCE_TIME: 500,
```

Comment:
Usually I like to include the unit for time constants (e.g. append `_MS`) to make it clear, but we already don't do it for the other constants above, so not blocking on this.